modules/tinymce/src/plugins/imagetools/test/ts/browser/ImageToolsDisabledButtonsTest.ts

```diff
@@ -0,0 +1,96 @@
+import { UiFinder, Waiter } from '@ephox/agar';
+import { before, describe, it } from '@ephox/bedrock-client';
+import { McEditor, TinyDom, TinySelections } from '@ephox/mcagar';
+import { Attribute, SugarBody } from '@ephox/sugar';
+import { assert } from 'chai';
+
+import PromisePolyfill from 'tinymce/core/api/util/Promise';
+import ImagePlugin from 'tinymce/plugins/image/Plugin';
+import ImageToolsPlugin from 'tinymce/plugins/imagetools/Plugin';
+import Theme from 'tinymce/themes/silver/Theme';
+
+describe('browser.tinymce.plugins.imagetools.ImageToolsDisabledButtonsTest', () => {
+
+  const settings = {
+    plugins: 'image imagetools',
+    toolbar: 'editimage fliph flipv rotateleft rotateright',
+    base_url: '/project/tinymce/js/tinymce',
+    height: 900
+  };
+
+  const editImageButtonSelector = '[role="toolbar"] button[title="Edit image"]';
+  const flipHorizontallyImageButtonSelector = '[role="toolbar"] button[title="Flip horizontally"]';
+  const flipVerticallyImageButtonSelector = '[role="toolbar"] button[title="Flip vertically"]';
+  const rotateCounterClockwiseImageButtonSelector = '[role="toolbar"] button[title="Rotate counterclockwise"]';
+  const rotateClockwiseImageButtonSelector = '[role="toolbar"] button[title="Rotate clockwise"]';
+
+  const assertImageToolsButtonsExist = (editor) => {
```

Comment:
Types please

Reply:
I don't think we need this helper function, the buttons can be assumed to be present because of the editor config (and we assert their enabled/disabled state anyway, which also asserts they are present).

Reply:
Good point and if they aren't visible then the assertions to check the state should fail anyways.

- [ ] Remove this assertion as it's unnecessary